Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemController.java

```diff
@@ -54,10 +56,15 @@
 
     @GetMapping
     public List<ProblemItem> getProblems(@RequestParam(value = "tags", required = false) String tagsSplitByCommas,
```

Comment:
Does this mean that you can put `String[] tags` here instead of `String tagsSplitByCommas` ?

Try it and tell me the result